packages/affiliates/payouts/2021-02-08_2021-02-15.json

```diff
@@ -1,80 +0,0 @@
-{
```

Comment:
why was this file removed?

Reply:
good question. i think this was added accidentally in a past PR!

Reply:
actually looking at it now, i think it was committed to the wrong directory, because its not in payouts/devmining where it should be. thanks for noticing this. will move it rather than delete